README.md

```diff
@@ -150,6 +149,22 @@ We compute the median accuracy of the last 20 checkpoints in the paper, this is
 # The format is JSON so you can either see its content as a text file or process it to your liking.
 ```
 
+## Adding datasets
+You can add custom datasets into the codebase by taking the following steps:
+
+1. Add a function to acquire the dataset to `scripts/create_datasets.py` similar to the present ones, e.g. `_load_cifar10`. 
+You need to call `_encode_png` to create encoded strings from the original images.
+It should return a dictionary of the format 
```

Comment:
What does "It" refer to?

Reply:
I have changed it to be more clear. Now says "The created function" instead of "It"